crates/uv-auth/src/lib.rs

```diff
@@ -3,18 +3,18 @@ use std::sync::{Arc, LazyLock};
 use tracing::trace;
 use url::Url;
 
+pub use auth_index::{AuthIndex, AuthIndexes, AuthPolicy};
 use cache::CredentialsCache;
 pub use credentials::Credentials;
 pub use keyring::KeyringProvider;
 pub use middleware::AuthMiddleware;
-pub use policy::{AuthPolicy, UrlAuthPolicies};
 use realm::Realm;
 
+mod auth_index;
```

Comment:
I'd probably call this `indexes` to match the existing naming scheme

Reply:
Yeah this naming isn't ideal, but since we can't depend on `IndexLocations` here without a circular crate dependency, the `AuthIndex` is used in the context of `IndexLocations` to construct an `AuthIndexes`.

A solution is to use the fully qualified name `uv_auth::index::Index` in the context of `IndexLocations`. So I'll do that.